RTCMultiConnection.js

```diff
@@ -2217,7 +2217,17 @@
             for (var i = 0; i < labels.length; i++) {
                 var label = labels[i];
                 if (connection.streams[label]) {
-                    peer.removeStream(connection.streams[label].stream);
+                    try {
+                        peer.removeStream(connection.streams[label].stream);
+                        // Fallback - on Firefox removeStream has not been implemented yet
+                    } catch (e) {
+                        // Force stop all the tracks
+                        var stream = connection.streams[label].stream;
+                        var tracks = stream.getTracks();
+
+                        for (var k in tracks)
+                            tracks[k].stop();
```

Comment:
[`removeStream`](http://www.rtcmulticonnection.org/docs/removeStream/) is not meant to stop the stream. Though it has **optional** `{stop:true}` parameter as well.

```javascript
connection.removeStream({
     local: true,
     stop: true
});;
```

We may need to modify SDP, remove media `ssrc` lines, and renegotiate both peers, and manually fire `rtcPeerConnection.onremovestream` for remote parties.

In normal `switch-stream` cases, we can either use [`applyConstraints `](https://github.com/muaz-khan/RTCMultiConnection/tree/master/RTCMultiConnection-v3.0#applyconstraints) or `rtpSender.replaceTrack`. First one makes more sense and easier to use (by changing `sourceId` or resolutions etc.).